src/_actions/officeLocations.actions.js

```diff
@@ -15,14 +15,21 @@ const {
   LOAD_COUNTRIES_FAILURE,
   UPDATE_ANDELA_CENTRE_REQUEST,
   UPDATE_ANDELA_CENTRE_SUCCESS,
-  UPDATE_ANDELA_CENTRE_FAILURE
+  UPDATE_ANDELA_CENTRE_FAILURE,
+  LOAD_OFFICE_BLOCK_REQUEST,
+  LOAD_OFFICE_BLOCK_SUCCESS,
+  LOAD_OFFICE_BLOCK_FAILURE,
+  CREATE_OFFICE_BLOCK_REQUEST,
+  CREATE_OFFICE_BLOCK_SUCCESS,
+  CREATE_OFFICE_BLOCK_FAILURE
 } = constants;
 
 export const loadOfficeLocations = (pageNumber, limit) => (dispatch) => {
   dispatch({ type: LOAD_LOCATIONS_REQUEST });
-
+  const url = (pageNumber && limit) ? `andela-centres/?page=${pageNumber}&page_size=${limit}`
```

Comment:
Can this be `(pageNumber || limit)` instead of `(pageNumber && limit)`

Reply:
No, it can't be because if one is not defined the request returns an error as the undefined parameter will result in` undefined` and the request looks like `andela-centres/?page=${pageNumber}&page_size=undefined`.